prov/ansible/execution.go

```diff
@@ -330,6 +332,17 @@ func (e *executionCommon) setEndpointCredentials(kv *api.KV, host, instanceID, c
 		if found && privateKey != "" {
 			conn.privateKey = config.DefaultConfigTemplateResolver.ResolveValueWithTemplates("host.privateKey", privateKey).(string)
 		}
+
+		found, port, err := deployments.GetInstanceCapabilityAttribute(e.kv, e.deploymentID, host, instanceID, "endpoint", "port")
+		if err != nil {
+			log.Debugf("[Warning] failed to get port attribute from capability endpoint with error: %+v", err)
```

Comment:
I think this should also be logged in deployment logs

Reply:
Same for the other debug log bellow

Reply:
Yes, maybe we should return other errors around user, password and private key ?